Answer q's

rodenbe4 opened the pull request:
I refactored answer_question() so that all of the return statements are strings

rodenbe4 committed:
added answer_question() to Team class

Team now has the functionality of answering questions, but I still need to add it to the interface.  Also still waiting on Landmark class to be merged as well

rodenbe4 committed:
Merge branch 'master' into answer-q's

rodenbe4 committed:
added answer_question functionality

-added unit tests for answer_question()
-fully implemented answer_question() in Team class
-added command to escavenge()
-removed some excess parentheses from if statements and while loops in escavenge()

rodenbe4 committed:
refactored answer_question

changed to all text-based return statements

tekkeon commented on team.py:
```diff
@@ -40,4 +42,20 @@ def edit_password(self, input):
+        except IndexError:
+            return "Not at a valid landmark"
+        ret_string = ""
+        if check_landmark.verify_answer(user_answer):
```
if check_landmark was created in the try block, can it still be accessed here? I don't know how the scope works in python so this may be valid, but I figured I'd ask

tekkeon commented on team.py:
```diff
@@ -40,4 +42,20 @@ def edit_password(self, input):
\ No newline at end of file
+        return ret
+
+    def answer_question(self, user_answer):
```
user_answer is not a string, but rather a list of strings, so to get the actual answer, you'd probably want to do this:

def answer_question(self, input):
user_answer = input[0]

rodenbe4 committed:
fixed issue with parameters

I changed the parameter of answer_question() to account for the new input type.  Everything should be good to go now

rodenbe4 committed:
hmm

for some reason, my latest commit doesn't have the changes that were made to the escavenge and gamemaker classes, so I'm re-committing to see if I can fix that

rodenbe4 committed:
let's try one more time

ugh

rodenbe4 committed:
I swear to god this better be the last time

for realsies

rodenbe4 committed:
I swear this is the last time

I've only been doing this so I can get rid of the merge conflicts. Ignore all of my other commits pleasethankyousomuch
